lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -871,7 +871,7 @@ fc_extras
 
     CM_PARSE = re.compile(  r'''
                                 (?P<name>([\w_]+\s*:\s+)+)
-                                (?P<method>[\w_\s]+(?![\w_]*:))\s*
+                                (?P<method>[\w_\s]+(?![\w_]*\s*:))\s*
```

Comment:
> Performance?

Yep. As I said, since the expressions being matched are short, I don't think it will make a huge difference either way.